src/app/features/Chanels/redux/reducers/channelsReducer.js

```diff
@@ -0,0 +1,20 @@
+import { GET_CHANNELS } from '../actionsTypes';
+
+const initialState = {
+  channels: [],
```

Comment:
will channelsState store anything except channels list? If not, make channelsState an Array.

Reply:
So that it will be just 
`initialState = []`